dev_readme.md

```diff
@@ -1,30 +1,78 @@
 # How to start website
 
-#### 1. Download and install [Docker](https://www.docker.com/products/docker-desktop) for your operating system.
+// TODO: Merge into main readme. 
 
-#### 2. Clone the repository into your local directory.
+If you're interested in building and running the AutSPACE platform, read on!
 
-  1. **Fork** the repository by clicking on the Fork icon at the top right hand corner of the page. 
-  2. Above the list of files, click the green **Code** button.
-  3. Clone the repository into your local directory. 
-  4. Open terminal to the project directory.
-  5. Type `git clone`, and then paste the URL you copied earlier. It will look like this, with your GitHub username instead of YOUR-USERNAME:
-  >$ `git clone https://github.com/YOUR-USERNAME/AutSPACEs`
-  7. Press enter. Your local clone will be created. 
-  
-#### 3. Create the .env file in config directory using the .env.template. Copy that template and follow the instructions with the README to generate the Django secret key.
-  
-  1. Go into "config" directory - find file called `env.template`
-  2. Make a copy of the template and rename it as `.env` within that same directory
-  3. Within that `.env` file, generate a `DJANGO_SECRET_KEY` following instructions defined in the file 
+## Setup
+Before starting, please check that the following minimum requirements have been completed.
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop). (Latest stable release, version 20.10.5 or greater)
+2. Install [Docker Compose](https://docs.docker.com/compose/install/). (Latest stable release, version 1.28.5 or greater)
+3. Create an [Open Humans](https://www.openhumans.org/) account.
+4. Some version of Python 3.x.x.
+
+## Installation
+Project installation can be tricky, but like all things, progress is made one step at a time. 
+
+1. Clone the repository at [AutSPACEs](https://github.com/GeorgiaHCA/AutSPACEs)
+2. Within the `config` directory, copy `.env.template` to `.env`. 
+  Note: ***DO NOT UPLOAD THESE .ENV FILES ONTO GITHUB*** It's the equivalent of posting your passport ID on Twitter. Don't do it.
+
+### Open Humans Project Creation
+This step is a little tricky, but you're almost there. 
+
+1. Login to [OpenHumans](openhumans.org). 
+2. Navigate to [Manage Project](https://www.openhumans.org/direct-sharing/projects/manage/).
+3. Create a new Oauth2 Data Request Project on Open Humans. 
+4. Populate the required fields as directed. 
```

Comment:
I think this is likely to be a little tricky for users - I wonder if we could provide a full template for what to put in

Reply:
This step of creating an OpenHumans is the biggest hurdle. 
I think there needs to be a separate readme or maybe another tutorial that defines what fields need to be populated or how to go about creating a new project (step by step) strictly for development.